config/presets/identity/email-sender.json

```diff
@@ -2,7 +2,7 @@
   "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^0.0.0/components/context.jsonld",
   "@graph": [
     {
-      "comment": "This value can be used to set up an actual e-mail client",
+      "comment": "The default configuration does not contain credentials for an email client. In production systems, you likely want to set up your own.",
```

Comment:
I'm assuming this will not result in a crash? So it will just log an error that the email cannot be sent? Or will it crash?

Reply:
Somewhere in between. It will throw an error, but the only time this is used is on the "forgot password" page, of which the handler already catches all errors. So this will result in an error being displayed on that page when the submit button is pressed.